recipes/gmpyinfr_telegram/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "gmpyinfr_telegram" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gmpyinfr_telegram-{{ version }}.tar.gz
+  sha256: ddd49f907d12beb72d529fe2c984c2bf81aeaac325451ecd8a5ddfb1c285d21f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or osx or py<3.6 or py>3.8]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

You're right. Thank you.